src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  background-color: $color-neutral;
+  padding: 16px;
+  border: 1px solid $color-border;
+  border-radius: 5px;
+  justify-content: space-between;
+  transition: transform, color, 300ms;
+
+  &__title {
+    color: $color-main;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__top {
+    width: 160px;
+    height: 134px;
+    display: block;
+    padding-top: 16px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-secondary;
+    align-self: start;
+    padding-top: 4px;
+    margin: 0;
+  }
+
+  &__rates {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+    padding-bottom: 12px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
```

Comment:
Use for your elements only `margin-top` or `margin-bottom` to avoid margin collapse https://www.freecodecamp.org/news/what-is-margin-collapse-and-how-to-avoid-it/